v-next/hardhat/example.config.ts

```diff
@@ -1,22 +1,49 @@
-import {
-  HardhatUserConfig,
-  configVariable,
-  overrideTask,
-  task,
-} from "./src/config.js";
-
-const exampleTaskOverride = overrideTask("example")
-  .setAction(async (args, _hre, runSuper) => {
+import type { HardhatUserConfig } from "./src/config.js";
```

Comment:
Just a note, this config to play around with v3 can now be done in `example-project`.